features/step_definitions/custom_web_steps.rb

```diff
@@ -142,6 +142,10 @@ def take_screenshots_with_login
   page.driver.browser.switch_to.alert.dismiss
 end
 
+And /^I should not see any alert$/ do
+  expect { page.driver.browser.switch_to.alert }.to raise_error(Selenium::WebDriver::Error::NoAlertPresentError)
+end
```

Comment:
It's not nice and would get a bit nicer with a custom matcher, but it should work.